apps/camera_pipe/camera_pipe.cpp

```diff
@@ -15,13 +17,18 @@ Expr avg(Expr a, Expr b) {
 }
 
 Func hot_pixel_suppression(Func input) {
+    // hot pixel maximum
     Expr a = max(max(input(x-2, y), input(x+2, y)),
                  max(input(x, y-2), input(x, y+2)));
-    Expr b = min(min(input(x-2, y), input(x+2, y)),
-                 min(input(x, y-2), input(x, y+2)));
+    // cold pixel minimum
+    // Expr b = min(min(input(x-2, y), input(x+2, y)),
```

Comment:
I think I went back and forth on whether cold pixel removal was a good idea in the original FCam. It doesn't really affect performance, so I'd just remove it.